.github/ISSUE_TEMPLATE.md

```diff
@@ -17,5 +17,7 @@ If the node is "stuck" during sync or giving "block checksum mismatch" errors, p
 
 <!-- What type of machine are you observing the error on (OS/CPU and disk type)? -->
 
+<!-- For the GUI-related issue on Linux provide names and versions of a distro, a desktop environment and a graphical shell. -->
```

Comment:
What is a graphical shell? Maybe this should be and/or?

Reply:
Clarifying here won't explain it to someone submitting an issue. The comment itself needs to be clearer.

Perhaps "graphical shell" is a familiar term for GNOME users, but most KDE users will never encounter it, since almost all use Plasma+KWin.

"*(if relevant)*" sounds good enough.